docs/MONGO_DB_SETUP.md

```diff
@@ -0,0 +1,53 @@
+
+# Mongo_db setup
+
+1. First thing you need to go to this url [mongo_db login](https://account.mongodb.com/account/login).
+
+  
+
+2. After login (or register incase you donot have account) there will be a page in top of it **Deploy a cloud database** with 3 options (Serverless - Dedicated - Shared) you will choose **Shared** option
```

Comment:
Its either `do not` or its `don't`.

Please run a spell check

Reply:
@amir-ammar

Reply:
done